winter/web/autodiscovery.py

```diff
@@ -0,0 +1,23 @@
+from typing import List
+
+from winter.core import Component
+from winter.core.module_discovery import get_all_classes
+from winter.core.module_discovery import import_recursively
+from winter.web.routing import Route
+from winter.web.routing import get_route
```

Comment:
I think tests failed due to these absolute import within a single package (winter.web)